[BUG] NATSS CCP test can flake on minute boundary

**Describe the bug**
Got a flake of this test during [the v0.5.0 build](https://gubernator.knative.dev/build/knative-prow/logs/ci-knative-eventing-auto-release/1113170595467497472/):

```
I0402 20:11:22.956]     --- FAIL: TestAllCases/deleted_clusterChannelProvisioner (0.01s)
I0402 20:11:22.956]         table.go:122: Unexpected present *v1alpha1.ClusterChannelProvisioner /natss (-want +got):
I0402 20:11:22.956]             {*v1alpha1.ClusterChannelProvisioner}.ObjectMeta.DeletionTimestamp:
I0402 20:11:22.956]             	-: s"2019-04-02 20:11:14 +0000 UTC"
I0402 20:11:22.957]             	+: s"2019-04-02 20:11:13 +0000 UTC"
```

 **Expected behavior**
No flakes

 **To Reproduce**
Probably very difficult to repro since the timing must occur across minute boundary. Could add a sleep to force the issue.

 **Knative release version**
v0.5.0

 **Additional context**

--- a/contrib/natss/pkg/controller/clusterchannelprovisioner/reconcile_test.go
+++ b/contrib/natss/pkg/controller/clusterchannelprovisioner/reconcile_test.go
@@ -49,6 +49,7 @@ func init() {
 
 var (
 	truePointer = true
+	deletedTime = metav1.Now().Rfc3339Copy()
 )
 
 var mockFetchError = controllertesting.Mocks{
@@ -187,7 +188,6 @@ func getNewClusterChannelProvisioner(name string, reconcileKind string) *eventin
 
 func makeDeletedClusterChannelProvisioner() *eventingv1alpha1.ClusterChannelProvisioner {
 	c := makeClusterChannelProvisioner()
-	deletedTime := metav1.Now().Rfc3339Copy()
 	c.DeletionTimestamp = &deletedTime
 	return c
 }

